app/src/main/res/values/strings.xml

```diff
@@ -1229,4 +1229,10 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="quest_road_width_title">"What’s the width of this road here?"</string>
     <string name="quest_road_width_explanation">"The roadway width from curb to curb: This includes anything on the street surface like on-street parking or cycle lanes but excludes anything beyond it like cycle paths or parking beside the roadway."</string>
 
+    <string name="quest_seating_name_title">Does this place have seating?</string>
```

Comment:
```suggestion
    <string name="quest_seating_name_title">What kind of seating does this place have?</string>
```

Minor nitpick, my suggestion given the answers are the values like takeaway, indoor, outdoor rather than yes/no values.